db/chores/migrate_messages_to_scheduled_messages.rb

```diff
@@ -0,0 +1,91 @@
+class MigrateMessagesToScheduledMessages
+  def perform
+    execute("SELECT * FROM onboarding_messages").each do |row|
+      migrate_onboarding_message_to_scheduled_message(row)
+    end
+    execute("SELECT * FROM quarterly_messages").each do |row|
+      migrate_quarterly_message_to_scheduled_message(row)
+    end
+  end
+
+  private
+
+  def migrate_onboarding_message_to_scheduled_message(row)
+    # Migrate to new table
+    row["type"] = 0
+    values = ["created_at","updated_at","title","body","days_after_start","time_of_day","type","end_date","deleted_at"].map do |column|
+      sanitize(row[column])
+    end
+    scheduled_message_id = insert(
+      <<-SQL
+        INSERT INTO scheduled_messages (created_at, updated_at, title, body, days_after_start, time_of_day, type, end_date, deleted_at)
+        VALUES (#{values.join(",")})
+      SQL
+    )
+
+    # Update sent messages
+    execute(
+      <<-SQL
+        UPDATE sent_messages
+        SET message_id = #{scheduled_message_id}, message_type = NULL
+        WHERE message_id = #{row["id"]} AND message_type = 'OnboardingMessage'
+      SQL
+    )
+
+    # Update tags
+    execute(
+      <<-SQL
+        UPDATE taggings
+        SET taggable_id = #{scheduled_message_id}, taggable_type = 'ScheduledMessage'
+        WHERE taggable_id = #{row["id"]} AND taggable_type = 'OnboardingMessage'
+      SQL
+    )
+  end
+
+  def migrate_quarterly_message_to_scheduled_message(row)
+    # Migrate to new table
+    row["type"] = 1
+    row["time_of_day"] = "2000-01-01 12:00:00"
+    values = ["created_at","updated_at","title","body","days_after_start","time_of_day","type","end_date","deleted_at"].map do |column|
+      sanitize(row[column])
+    end
+    quarterly_message_id = insert(
+      <<-SQL
+        INSERT INTO scheduled_messages (created_at, updated_at, title, body, days_after_start, time_of_day, type, end_date, deleted_at)
+        VALUES (#{values.join(",")})
+      SQL
+    )
+
+    # Update sent messages
+    execute(
+      <<-SQL
+        UPDATE sent_messages
+        SET message_id = #{quarterly_message_id}, message_type = NULL
+        WHERE message_id = #{row["id"]} AND message_type = 'QuarterlyMessage'
+      SQL
+    )
+
+    # Update tags
+    execute(
+      <<-SQL
+        UPDATE taggings
+        SET taggable_id = #{quarterly_message_id}, taggable_type = 'ScheduledMessage'
+        WHERE taggable_id = #{row["id"]} AND taggable_type = 'QuarterlyMessage'
+      SQL
+    )
+  end
+
+  # Helpers
+
+  def execute(*args)
```

Comment:
not necessary but it might be good to put these into a shared place since you use them in two places 🌞